stock_cost_method_last/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Last Price Costing Method
+=========================
+
+This module adds a new *Costing Method*: *Last Price*. When this method is selected, the cost of the product is set to the last purchase price.
+
+Configuration
+=============
+
+On the form view of the product category, in the *Inventory Valuration* section, you can now select *Last Price* as *Costing Method*.
+
+Usage
+=====
+
+When you validate the incoming picking, the *Cost Price* of the product will be updated to the price of the related purchase order line (taking into account the currency, unit of measure and taxes).
```

Comment:
You should explain that if purchase module is not installed, it will take standard price.

Reply:
I agree that it something that could be clarified since the dependency with purchase is not explicit.